src/applications/personalization/profile360/actions/index.js

```diff
@@ -16,7 +16,7 @@ export function fetchHero() {
     const response = await getData('/profile/full_name');
 
     if (response.errors || response.error) {
-      dispatch({ type: FETCH_HERO_FAILED, hero: { response } });
+      dispatch({ type: FETCH_HERO_FAILED, hero: { errors: response } });
```

Comment:
Totally get having the state look like `hero.errors.errors...` is gross but since we aren't actually doing anything with the errors right now can we live with it? :-P I guess the alternative is to simply set a flag like `hero: {error: true}`!

Reply:
Changed it to your initial suggestion! Thanks for catching this 💯